src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -877,10 +877,12 @@ def _find_using_common_tenant(self, access_token, resource):
         tenants = client.tenants.list()
         for t in tenants:
             tenant_id = t.tenant_id
-            if not hasattr(t, 'display_name'):  # Available since 2018-06-01
-                t.display_name = ""
-            if hasattr(t, 'additional_properties'):  # remove this line once SDK is fixed
-                t.display_name = t.additional_properties['displayName']
+            # display_name is available since /tenants?api-version=2018-06-01,
+            # not available in /tenants?api-version=2016-06-01
+            if not hasattr(t, 'display_name'):
+                t.display_name = None
+            if hasattr(t, 'additional_properties'):  # Remove this line once SDK is fixed
+                t.display_name = t.additional_properties.get('displayName')
```

Comment:
Fall back to `None` if `displayName` is not present.

Reply:
What's the difference between None and empty string here ? Only impact the warning message ? It will print `00000000-0000-0000-0000-000000000000 ''` before ?

Reply:
> What's the difference between None and empty string here?

No difference.

> Only impact the warning message? It will print 00000000-0000-0000-0000-000000000000 '' before?

Yes.